### DISCUSSION TITLE
Help with Signed Messages

### DISCUSSION THREAD

**Discussion by JThomasewsky** [1200wd/bitcoinlib#336] [Q&A] [answered]
Hello i'm latinamerican so sorry for my bad english, i'm using bitcoinlib to try to see who's the signer of a message getting the address, the message and the signature, so idk how i can see this with bitcoinlib, this is my argument, i use this with bitcoin core, i think bitcoinlib have something like this but idk what's, if somebody can help me i'm gonna feel grated

verifymessage "address" "signature" "message"

**Comment by mccwdev** [comment, marked as answer]
Sorry, at the moment this is not possible, as it still needs to be implemented. The library can sign and verify transactions only.

See issue #288
